pkl-core/src/main/java/org/pkl/core/project/ProjectPackager.java

```diff
@@ -358,6 +359,7 @@ private List<Path> collectPackageElements(Project project, Package pkg) {
                 }
                 return true;
               })
+          .filter((it) -> !Files.isDirectory(it) && !isEmptyDirectory(it))
```

Comment:
Something is wrong with this condition. A test would help to catch it.

Reply:
Isn't there a test already available? 🤔
I even updated it...

Reply:
I assumed there wasn’t because the condition doesn’t look right. “Not a directory” implies “not an empty directory”.

Reply:
Yes, you're right. Something seems not correct here 🤔
Will check out tomorrow. Thanks for pointing out 🤝

Reply:
I found another solution which seems easier to understand 🙃 See https://github.com/apple/pkl/pull/330/commits/a0ee462f59b654b1ccaf403c5132a24057734815

Reply:
Your force-push broke the link

Reply:
The link I posted in my preview comment is the last commit (the force push) and for me it still works 🤔